applications/CompressiblePotentialFlowApplication/custom_operations/potential_to_compressible_navier_stokes_operation.h

```diff
@@ -0,0 +1,121 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Marco Antonio Zuñiga Perez
+//                   
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "operations/operation.h"
+
+namespace Kratos
+{
+  ///@addtogroup CompressiblePotentialFlowApplication
+  ///@{
+
+///@name Kratos Classes
+///@{
+
+/**
+ * @author Marco Antonio Zuñiga Perez
+ * @class PotentialToCompressibleNavierStokesOperation
+ * @ingroup CompressiblePotentialFlowApplication
+ * @brief This operation pass the nodal velocities from Potential model part as initial condition of a compressible 
+ * Navier Stokes model part in conservative variables form.
+*/
+class KRATOS_API(KRATOS_COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION) PotentialToCompressibleNavierStokesOperation : public Operation
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of Operation
+    KRATOS_CLASS_POINTER_DEFINITION(PotentialToCompressibleNavierStokesOperation);
+
+    /// Registry current operation
+    KRATOS_REGISTRY_ADD_PROTOTYPE("Operations.KratosMultiphysics.CompressiblePotentialFlowApplication", PotentialToCompressibleNavierStokesOperation)
+    KRATOS_REGISTRY_ADD_PROTOTYPE("Operations.All", PotentialToCompressibleNavierStokesOperation)
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Default constructor.
+    PotentialToCompressibleNavierStokesOperation() : Operation() {}
+
+    PotentialToCompressibleNavierStokesOperation(
+        Model& rModel,
+        Parameters OperationParameters);
+
+    /// Destructor.
+    ~PotentialToCompressibleNavierStokesOperation() override = default;
+    /// Copy constructor.
+    //TODO: Check. It is required by the registry
+    PotentialToCompressibleNavierStokesOperation(PotentialToCompressibleNavierStokesOperation const& rOther)  = delete;
```

Comment:
```suggestion

    /// Copy constructor.
    PotentialToCompressibleNavierStokesOperation(PotentialToCompressibleNavierStokesOperation const& rOther)  = delete;
```
Yes, it is required.